libr/debug/p/native/reg.c

```diff
@@ -1,7 +1,7 @@
 // included from p/debug_native.c
 // splitted for better reading/cleaning up
 
-static const char *r_debug_native_reg_profile(RDebug *dbg) {
+static char *r_debug_native_reg_profile(RDebug *dbg) {
```

Comment:
why removing 'const' here?

Reply:
supress warning casting const char * to char*:
....
char* (*reg_profile) (RDebug *dbg);
} RDebugPlugin;